parser.go

```diff
@@ -329,6 +331,20 @@ func (r *Rule) option(key item, l *lexer) error {
 			return fmt.Errorf("invalid reference definition: %s", refs)
 		}
 		r.References = append(r.References, &Reference{Type: refs[0], Value: refs[1]})
+	case "metadata":
+		nextItem := l.nextItem()
+		if nextItem.typ != itemOptionValue {
+			return errors.New("no valid value for metadata")
+		}
+		metas := strings.Split(nextItem.value, ", ")
```

Comment:
I think it might be a bit safer to either use a regexp to handle multiple spaces, or the no spaces case. 
metaSplitRE := regexp.MustCompile(`,\s*`) //global

metas := metaSplitRE.(nextitem.value, -1)

Reply:
thanks for pointint that, I modified the code,please check that.